pkg/fleet/env/env.go

```diff
@@ -108,13 +128,51 @@ func (e *Env) ToEnv() []string {
 	if e.RegistryAuthOverride != "" {
 		env = append(env, envRegistryAuth+"="+e.RegistryAuthOverride)
 	}
+	if len(e.ApmLibraries) > 0 {
+		b := strings.Builder{}
+		b.WriteString(envApmLibraries)
+		b.WriteString("=")
+		for l, v := range e.ApmLibraries {
+			b.WriteString(string(l))
+			if v != "" {
+				b.WriteString(":")
+				b.WriteString(string(v))
+			}
+			b.WriteString(",")
+		}
+		envVar := b.String()
+		env = append(env, envVar[:len(envVar)-1])
+	}
 	env = append(env, overridesByNameToEnv(envRegistryURL, e.RegistryOverrideByImage)...)
 	env = append(env, overridesByNameToEnv(envRegistryAuth, e.RegistryAuthOverrideByImage)...)
 	env = append(env, overridesByNameToEnv(envDefaultPackageInstall, e.DefaultPackagesInstallOverride)...)
 	env = append(env, overridesByNameToEnv(envDefaultPackageVersion, e.DefaultPackagesVersionOverride)...)
 	return env
 }
 
+func parseApmLibrariesEnv() map[ApmLibLanguage]ApmLibVersion {
+	apmLibraries := os.Getenv(envApmLibraries)
+	if apmLibraries == "all" {
+		return map[ApmLibLanguage]ApmLibVersion{
+			"java":   "latest",
+			"ruby":   "latest",
+			"js":     "latest",
+			"dotnet": "latest",
+			"python": "latest",
+		}
+	}
+	apmLibrariesVersion := map[ApmLibLanguage]ApmLibVersion{}
+	rest := apmLibraries
+	for rest != "" {
+		fmt.Println(rest)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏Leftover print